semantic-conventions/src/opentelemetry/semconv/model/semantic_convention.py

```diff
@@ -300,9 +300,12 @@ class SemanticConventionSet:
 
     debug: bool
     models: typing.Dict[str, BaseSemanticConvention] = field(default_factory=dict)
+    extended_models: typing.Dict[str, BaseSemanticConvention] = field(
+        default_factory=dict
+    )
     errors: bool = False
 
-    def parse(self, file):
+    def parse(self, file, ref_only=False):
```

Comment:
Maybe instead add `parse_into(self, file, models)`?

If you prefer keeping the boolean argument, please add `*` before to enforce using named arguments for it:

```suggestion
    def parse(self, file, *, ref_only=False):
```